test/e2e/cluster_logging_es.go

```diff
@@ -153,12 +162,20 @@ func checkElasticsearchReadiness(f *framework.Framework) error {
 			framework.Logf("After %v failed to get services proxy request: %v", time.Since(start), errProxy)
 			continue
 		}
+
+		ctx, cancel := context.WithTimeout(context.Background(), framework.SingleCallTimeout)
+		defer cancel()
+
 		body, err = proxyRequest.Namespace(api.NamespaceSystem).
+			Context(ctx).
 			Name("elasticsearch-logging").
 			Suffix("_cluster/health").
 			Param("level", "indices").
 			DoRaw()
 		if err != nil {
+			if ctx.Err() != nil {
+				framework.Failf("Failed to get cluster health from elasticsearch: %v", err)
```

Comment:
Generally `err` indicates that the request was cancelled, and which request it was. The `ctx.Err()` has less information. I'm going to go back through and make sure this is consistent.